core/src/test/java/org/mskcc/cbio/oncokb/util/FindRelevantAlterationsTest.java

```diff
@@ -131,6 +131,8 @@ public FindRelevantAlterationsTest(String hugoSymbol, String alteration, String
 //                {"MED12", "A22*", null, "1_33mut, Truncating Mutations"},
 //                {"MED12", "G22D", null, "1_33nontrunc, 1_33mut"},
 
+                // Check VUS mutations
+                {"BRD4", "N54K", null, "N54K, VUS Mutations"},
```

Comment:
This test is invalid since the BRD4 does not have VUS yet. We could comment out this section and update it when we actually have the data.
Also this is should be `Variants of Unknown Significance`